docs/inso-cli/cli-command-reference/OAS-spec.yml

```diff
@@ -0,0 +1,618 @@
+---
+layout: article-detail
+title:  OpenAPI to Kong Spec
+category: "CLI Command Reference"
+category-url: inso-cli
+---
```

Comment:
```suggestion
```
I _think_ you can remove this. Right now, it's trying to generate HTML: https://insomnia-docs-git-ga-04-21-teaminsomnia.vercel.app/inso-cli/cli-command-reference/OAS-spec.yml